scripts/check_imports.py

```diff
@@ -0,0 +1,260 @@
+#!/usr/bin/env python3
```

Comment:
This should also only be run in the paths, `components`, `pipelines`, `third_party/components`, `third_party/pipelines`.